### PR TITLE
Minor improvements to make dev easier

### PR DESCRIPTION
**Pull request by Spirita**
- Add chat commands to force the weather
- Update deprecated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+Snowdrift is a server friendly mod that add weather effects with their sound effects.
+It avoid abm and useless calculations.
+
+
+##2 Current version
+
+Current :
+v0.7.0
+Tested with Minetest 0.4.13
```
Comment (bell07) [inline README.md]: You should update your Minetest. Current stable is 0.4.16. Many uses daily builds...
I tested the mod on 0.4.16 and it still works

File: lib/utils.lua [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+]]
+
+
+-- Function on vectors
+
+--[[ snowdrift.addvectors((v1, v2)
+Add the two given vectors.
+return the result as a vector.
+]]
+function snowdrift.addvectors(v1, v2)
```
Comment (bell07) [inline lib/utils.lua]: Just use existing `vector.add(v, x)`
See http://dev.minetest.net/vector
So you can remove the utils.lua file

File: src/calculation.lua [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+-- Function to calculate position
+-- ==============================
+
+--[[ snowdrift.ppos_for_player(player)
+Calculate the position to use for the given player.
+Return a table with a value for x, y and z.
+]]
+function snowdrift.ppos_for_player(player)
+	local ppos = player:getpos()
+	local pposy = math.floor(ppos.y) + 2 -- Precipitation when swimming
```
Comment (bell07) [inline src/calculation.lua]: vector.round(v) ??

File: lib/utils.lua (hunk 2 above)
Comment (Spirita) [inline lib/utils.lua]: done
I keep the utils.lua for the moment, i think use the code of paramt to create a particulespawner that chek a condition before spawn.

File: init.lua [hunk 4]
```diff
@@ -1,5 +1,6 @@
 -- Parameters
 
+weather = "default"
```
Comment (Spirita) [inline init.lua]: done

File: README.md (hunk 1 above)
Comment (Spirita) [inline README.md]: I use the depot of ubuntu, bad idea ?

File: src/calculation.lua (hunk 3 above)
Comment (Spirita) [inline src/calculation.lua]: I don't understand calculations from paramat.
I prefer don't modified it.

File: README.md (hunk 1 above)
Comment (bell07) [inline README.md]: I do not use ubuntu, but if I look to http://www.minetest.net/downloads/ the latest stable is available by "artful"  https://packages.ubuntu.com/artful/minetest
or daily build at https://code.launchpad.net/~minetestdevs/+archive/ubuntu/daily-builds/
I highly recommend to update your minetest at least to the latest stable, the most mods requires them in the meantime

**Comment by Spirita** [comment]
I have tested it in singleplayer but I have realize that I have bug the sound for multiplayer, wait for the v0.7.1

**Pull request merged**